src/test/java/hw3/po/DifferentElements.java

```diff
@@ -0,0 +1,121 @@
+package hw3.po;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.Select;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class DifferentElements extends BasePage {
+
+    @FindBy(css = "label.label-checkbox")
+    private List<WebElement> checkboxes;
+
+    @FindBy(css = "div.checkbox-row > label.label-radio")
+    private List<WebElement> radios;
+
+    @FindBy(css = "select.uui-form-element")
+    private WebElement webColor;
+    private Select color;
+
+    @FindBy(css = "ul.panel-body-list > li")
+    private List<WebElement> logs;
+    private List<String[]> logsData;
+
+    public DifferentElements (WebDriver driver){
+        this.driver = driver;
+        this.URL = driver.getCurrentUrl();
+        PageFactory.initElements(driver, this);
```

Comment:
May be it will be better to move invocation of the `PageFactory` to the `BasePage` class?